packages/contracts/src/utils/StringToArrayUtils.sol

```diff
@@ -20,7 +20,7 @@ library StringToArrayUtils {
         for (uint i = 0; i < parts.length; i++) {
             parts[i] = slicee.split(delim).toString();
         }
-        require(parts.length == 3, "Invalid kid|iss|azp strings");
+        require(parts.length == 2, "Invalid kid|iss strings");
```

Comment:
@wshino Can you please swap this sequence to have `iss|kid` as mentioned in the previous comment?